pom.xml

```diff
@@ -70,6 +70,7 @@
 			<groupId>com.google.code.gson</groupId>
 			<artifactId>gson</artifactId>
 			<version>2.10.1</version>
+			<optional>true</optional>
```

Comment:
Remove this. Because:
1. This will be a breaking change.
2. This will disrupt the kick-starting for novice users.
```suggestion
```

Reply:
Sure. Are you right. I'll provide the adjustment for that. Thanks